conftest.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+from typing import Dict
+
+from _pytest.reports import TestReport
+
+from src.pytest_merge_xml.merge_xml import modify_xml
+
+
+class XMLReport:
```

Comment:
Add some docstring:
Add a docstring. Something like:
```suggestion
class XMLReport:
    '''Class that collect test items and update and existing XML file with the new test results'''
    
```

Reply:
Done!